### DISCUSSION TITLE
Option to turn off or dim the background when screen scaling is used

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#23002] [Ideas] [answered]
#17611 is a similar discussion I saw, but I don't think it's quite the same as my suggestion. I'd like to see an option to turn off or dim the background when screen scaling is used, because when I'm playing taiko, my eyes are frequently distracted by the background, making it difficult for me to focus on the center of the screen.

![image](https://user-images.githubusercontent.com/43358824/227831721-7333d40f-e769-4729-b610-960f8ca50d0a.png)

**Comment by cdwcgt** [comment]
I think it is duplicated with my vote
please vote in https://github.com/ppy/osu/discussions/20839

**Reply by mouzedrift** [reply]
I believe the results are clear, and we are unlikely to receive much more feedback on GitHub now that the poll has been running for some time, and I don't think opening a PR would hurt since you appear to have already implemented a dim setting for the screen scale background.

![image](https://user-images.githubusercontent.com/43358824/228700150-f0cefb34-fc2c-4761-bbaf-3be9a5d1231d.png)

**Comment by Moltanicaa** [comment]
https://github.com/ppy/osu/discussions/17611

**Reply by mouzedrift** [reply]
Yeah, that's the first thing I mentioned in this discussion.

**Comment by cdwcgt** [comment, marked as answer]
See https://github.com/ppy/osu/pull/23038
